inception-html-editor/src/main/java/de/tudarmstadt/ukp/inception/htmleditor/annotatorjs/resources/annotator-full.js

```diff
@@ -269,7 +269,10 @@
     node = root;
     for (_i = 0, _len = steps.length; _i < _len; _i++) {
       step = steps[_i];
-      _ref1 = step.split("["), name = _ref1[0], idx = _ref1[1];
+      step.split("[");
```

Comment:
Generated file. No refactoring needed.

Reply:
Would You suggest that I shall close this PR also?

Reply:
No, the other changes are fine. Just the refactoring in this file (annotator-full.js) does not make sense as this file is not written or read by humans, but generated (coffeescript -> javascript). The rest of the pull request is fine.

Reply:
Sir, I have looked into the problem and it seems my commit from Eclipse had loaded two files into this single branch. The error or issue you have found was the other pull request which was closed based on our discussion in that branch with removing the comma operator earlier in the previous week. As of now, I am facing an error "404 page not found" to remove that annotator.js file from this branch. Please let me know what I could do.